packages/react-core/src/helpers/ouia.ts

```diff
@@ -19,6 +19,30 @@ export function getOUIAProps(componentType: string, id: OuiaId | undefined, ouia
   return {
     'data-ouia-component-type': `PF4/${componentType}`,
     'data-ouia-safe': ouiaSafe,
-    'data-ouia-component-id': id === undefined ? uid++ : id
+    'data-ouia-component-id': id === undefined ? `OUIA-${componentType}-${++uid}` : id
   };
 }
+
+const ouiaIdByRoute: any = {};
```

Comment:
while i appreciate more structural/to the point ouia ids, 
i fear that better autogeneration will be our worst impediment for correct usage

ideally people use database ids, or action identifies to make up those ids,
if the autogenerated ids seem good enough instead of indicating for certain that one has to set a correct id,
i fear a lot of missuse/missunderstanding coming our way

the intent of the ouia id is to identify objects/action in those objects in a way that matches aplication semantics

i would like to see a part of the autogenerated ouia id name to indicate its autogenerated and not matching intent/object-identities to at least ensure its not wrongly used/understood

Reply:
Ok, then how about i change the prefix to `OUIA-Generated-` ? i.e. `OUIA-Generated-Button-primary-1`

Reply:
That's definitely a good idea, such a indication would help to trigger some headscratching